src/altinn-app-frontend/src/components/advanced/AddressLabel.tsx

```diff
@@ -25,13 +25,16 @@ export const AddressLabel = ({
   return (
     <label className='a-form-label title-label' htmlFor={id}>
       {label}
-      {required ||
-      readOnly ||
-      labelSettings?.optionalIndicator === false ? null : (
+      {required && !readOnly &&
+        <span>
+        {` ${getLanguageFromKey('form_filler.required_label', language)}`}
+      </span>
+      }
+      {!required && !readOnly && labelSettings?.optionalIndicator &&
         <span className='label-optional'>
           {` (${getLanguageFromKey('general.optional', language)})`}
         </span>
-      )}
+      }
```

Comment:
This should probably also reuse the new `OptionalIndicator` and `RequiredIndicator` components?

Reply:
Yes 🤦